src/patternfly/components/Nav/nav.scss

```diff
@@ -1,169 +1,118 @@
-
-  // color
-  --pf-c-nav__tertiary-list-link--Color: var(--pf-global--Color--dark-100);
-  --pf-c-nav__tertiary-list-link--hover--Color: var(--pf-global--link--Color);
-  --pf-c-nav__tertiary-list-link--active--Color: var(--pf-global--link--Color);
-  --pf-c-nav__tertiary-list-link--focus--Color: var(--pf-global--link--Color);
-
-  // after
-  --pf-c-nav__tertiary-list-link--after--Height: #{pf-size-prem(3px)};
-  --pf-c-nav__tertiary-list-link--hover--after--BackgroundColor: var(--pf-global--link--Color);
-  --pf-c-nav__tertiary-list-link--active--after--BackgroundColor: var(--pf-global--link--Color);
-  --pf-c-nav__tertiary-list-link--focus--after--BackgroundColor: var(--pf-global--link--Color);
-
-  // Subnav links
-  --pf-c-nav__subnav--MarginTop: var(--pf-global--spacer--sm);
+  // Link
+  --pf-c-nav__link--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-nav__link--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-nav__link--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-nav__link--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-nav__link--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-nav__link--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-nav__link--Color: var(--pf-global--Color--dark-100);
+  --pf-c-nav__link--hover--Color: var(--pf-global--Color--dark-100);
+  --pf-c-nav__link--active--Color: var(--pf-global--Color--dark-100);
+  --pf-c-nav__link--focus--Color: var(--pf-global--Color--dark-100);
+  --pf-c-nav__link--BackgroundColor: transparent;
+  --pf-c-nav__link--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-200);
+  --pf-c-nav__link--active--BackgroundColor: var(--pf-c-nav__link--hover--BackgroundColor);
+  --pf-c-nav__link--focus--BackgroundColor: var(--pf-c-nav__link--hover--BackgroundColor);
+  --pf-c-nav__link--m-current--Color: var(--pf-global--Color--dark-100);
+  --pf-c-nav__link--m-current--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-c-nav__link--m-current--after--BorderWidth: var(--pf-global--BorderWidth--xl);
+
+  // Link ::before borders
+  --pf-c-nav__link--before--BorderColor: var(--pf-global--BorderColor--300);
+  --pf-c-nav__link--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-nav__link--before--BorderTopWidth: 0;
+  --pf-c-nav__link--before--BorderBottomWidth: 0;
+
+  // Link ::before borders
+  --pf-c-nav__link--after--BorderColor: var(--pf-global--active-color--400);
+  --pf-c-nav__link--after--BorderWidth: var(--pf-global--BorderWidth--xl);
+  --pf-c-nav__link--after--BorderBottomWidth: 0;
+  --pf-c-nav__link--after--BorderLeftWidth: 0;
+
+  // Nav list
+  --pf-c-nav__list__link--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-nav__list__link--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-nav__list__link--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-nav__list__link--PaddingLeft: var(--pf-global--spacer--lg);
```

Comment:
The horizontal padding for links should be `var(--pf-global--spacer--md)` < `$pf-global--breakpoint--md` to match the overall spacing changes that happen with the page layout between desktop and mobile. The left edge of the nav should align with the left edge of the hamburger on mobile.